packages/cli/src/__tests__/commands/platform/Platform.test.ts

```diff
@@ -0,0 +1,15 @@
+import { isError } from '@prisma/internals'
+
+import { PlatformCommand } from '../../../platform/PlatformCommand'
+
+describe('using cli', () => {
+  it('should not work without the --early-access-feature flag', async () => {
+    console.log(1)
+    const result = await PlatformCommand.new().parse([])
+    const resultIsError = isError(result)
+    expect(resultIsError).toBeTruthy()
+    if (resultIsError) {
+      expect(result.message).toMatch('This feature is currently in Early Access.')
+    }
```

Comment:
```suggestion
  await expect(
    PlatformCommand.new().parse([]),
  ).rejects.toMatchInlineSnapshot(``)
```

Can be simplified, it's best to not use a conditional
Alternatively, it can be a "toThrow()" with a matching message

Reply:
The conditional is to satisfy TS.

Reply:
Your suggestion assumes an error is thrown, but it is actually returned. Are you suggesting to throw an error instead of returning it?

Reply:
@Jolg42 I'll merge and we can of course continue changing things (transition to a throw and update this test, etc.)

Reply:
We should have some consistency in the CLI, we use throw indeed.